lib/tessel/deploy.js

```diff
@@ -528,6 +604,21 @@ actions.resolveBinaryModules = function(opts) {
   });
 };
 
+actions.resolveBinaryModules.readGypFileSync = function(gypfile) {
+  var python = process.env.PYTHON || 'python';
+  var program = `import ast, json; print json.dumps(ast.literal_eval(open("${gypfile}").read()));`;
+  var decoder = new StringDecoder('utf8');
+  var result = cp.spawnSync(python, ['-c', program]);
+  var output = result.output;
+
+  return output.reduce((accum, buffer) => {
+    if (buffer) {
+      accum += decoder.write(buffer);
+    }
+    return accum;
```

Comment:
Two things really stand out for me: 

- The addition of spawnSync
- Template strings really make a difference